src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction.mdx

```diff
@@ -15,428 +15,108 @@ redirects:
+        Future proof
       </td>
 
       <td>
-        ✅ (Must be non-nested and 64 or less elements)
+        While OpenTelemetry has already come a long way, it seems poised to grow in adoption thanks to its large active community, industry support, and open governance model. While we can't see the future, OpenTelemetry is the most likely open source winner in the observability industry.
       </td>
     </tr>
   </tbody>
 </table>
 
-### Logs [#logs-ingest]
+## OpenTelemetry or New Relic instrumentation? [#choice]
 
-New Relic offers support for the OTLP ingest of log signals. The maturity of the upstream specification is [stable](https://github.com/open-telemetry/opentelemetry-specification/blob/87a5ed7f0d4c403e2b336f275ce3e7fd66a8041b/specification/versioning-and-stability.md#stable).
+In many cases, there is overlap between features and components available from OpenTelemetry and from New Relic. For example, [OpenTelemetry APM monitoring](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/apm-monitoring/opentelemetry-apm-intro/) mirrors [New Relic APM agents](/docs/apm/new-relic-apm/getting-started/introduction-apm/), and [infrastructure monitoring with the OpenTelemetry collector](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-intro/) mirrors the capabilities of the [New Relic infrastructure agent](/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring/).
 
-OpenTelemetry logs are compatible with logs in New Relic. The OpenTelemetry logs optionally include attributes (name-value pairs) and resource attributes that map directly to dimensions you can use to facet or filter log data with queries. OpenTelemetry log metadata (for example, `name`, `severity_text`, and `trace_id`) also map directly to dimensions on New Relic's <InlinePopover type="logs" /> capabilities. We currently support all OpenTelemetry log message types.
+We recommend that you explore both New Relic and OpenTelemetry options. With New Relic instrumentation, there are inherent advantages to developing instrumentation and platform features which work together, and New Relic integrations tend to work better out of the box. On the other side, OpenTelemetry offers an unparalleled degree of flexibility and control, but this sometimes requires additional research and effort to achieve the desired outcome.
 
-For more details, see the [logs information in our best practices guide](/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts#logs).
+For a closer look at the differences between OpenTelemetry and New Relic instrumentation, see our [detailed comparison](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison).
```

Comment:
Here's the link to only link to the comparison page previously available at "OpenTelemetry -> Overview -> Feature comparison". I considered deleting it right now, but the content on this page needs some needs some attention, but there are occasional valuable bits of insight, so I figured its better to keep it available but less prominent. We need to revisit the content in a followup.

Reply:
Discussed in meeting and decided to remove the comparison and this link to it and come back to it later.